src/admin-panel/index.js

```diff
@@ -84,7 +83,7 @@ async function createLevel() {
     alert(err);
   } finally {
     removeLoadingState();
-    button.textContent = 'Create Level';
+    document.querySelector('.submit-button').textContent = 'Create Level';
```

Comment:
Why was this change needed?

Reply:
because of my mistake in previous PR

Reply:
On line 63, we already have a reference. `const button = document.querySelector('.submit-button');`, so why are we not using it here?

Reply:
because it's scope is limited to the try block